registry/auth/token/token.go

```diff
@@ -34,6 +34,7 @@ var (
 // ResourceActions stores allowed actions on a named and typed resource.
 type ResourceActions struct {
 	Type    string   `json:"type"`
+	Class   string   `json:"class"`
```

Comment:
`Class` describes a "class" of targets contained in the repository, whereas `Type` describes the type of the resource.